tests/translator/input/error_graphqlapi.yaml

```diff
@@ -0,0 +1,32 @@
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  NoAuthAPI:
+    Type: AWS::Serverless::GraphQLApi
+    Properties:
+      XrayEnabled: true
+      Tags:
+        Key1: Value1
+        Key2: Value2
```

Comment:
please let's add an empty line between resources, it's too overwhelming right now